### PR TITLE
Adapt tests for seat adjuster to work after importing to template

### PR DESCRIPTION
**Pull request by dennismeister93**
## Describe your changes

Adapt tests for seat adjuster to work after importing to template

### CODE REVIEW

File: examples/seat-adjuster/tests/unit/test_run.py [hunk 1]
```diff
@@ -15,56 +15,52 @@
...
         return_value=get_error_response(),
     ):
-        location = SeatLocation(row=1, index=1)
         get_position = get_sample_request_data()
-        response = await SeatService.MoveComponent(
-            location, BASE, get_position["position"]  # type: ignore
+        response = await vehicle.Cabin.Seat.Row1.Pos1.Position.set(
+            get_position["position"]
         )
         assert response == get_error_response()
```
Comment (BjoernAtBosch) [inline examples/seat-adjuster/tests/unit/test_run.py:65]: My impression is that we only test the mock with those tests ...
Real unit test should test calling e.g. `on_set_position_request_received` pasing a request and then mocks the speed get and checks if position set is triggered (if speed is 0) or an error is returned (if speed > 0 or position is out of range)

File: examples/seat-adjuster/tests/integration/integration_test.py [hunk 2]
```diff
@@ -13,13 +13,15 @@
...
-from asyncio import sleep
 
 import pytest
 
 from sdv.test.inttesthelper import IntTestHelper
 from sdv.test.mqtt_util import MqttClient
 
+# from asyncio import sleep
+
+
```
Comment (BjoernAtBosch) [inline examples/seat-adjuster/tests/integration/integration_test.py:24]: Remove

**Review by BjoernAtBosch — CHANGES_REQUESTED** [review]

**Review by BjoernAtBosch — CHANGES_REQUESTED** [review]

File: examples/seat-adjuster/src/vapp.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+from sdv.util.log import (  # type: ignore
+    get_opentelemetry_log_factory,
+    get_opentelemetry_log_format,
+)
+from sdv.vdb.reply import DataPointReply
+from sdv.vehicle_app import VehicleApp, subscribe_topic
+
+logging.setLogRecordFactory(get_opentelemetry_log_factory())
+logging.basicConfig(format=get_opentelemetry_log_format())
+logging.getLogger().setLevel("DEBUG")
```
[the remark this answers is not in the document]
Comment (dennismeister93) [reply examples/seat-adjuster/src/vapp.py:31]: unfortunately yes :(

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Pull request merged**